### PR TITLE
tasks' completion stats

### PR DESCRIPTION
**Pull request by vladimir-8**

### CODE REVIEW

File: cube/Task.js [hunk 1]
```diff
@@ -1,16 +1,80 @@
...
+  + DATE_PART('second', ${datetime})`
+
+
+const dateDiffInMinutes = (datetime) =>
+  `(${dateDiffInSeconds(datetime)}) / 60`
+
+
 cube(`Task`, {
-  sql: `SELECT id, type, done_after AS after, done_before AS before, status FROM public.task`,
+  sql: `SELECT id, type, done_after, done_before, status FROM public.task`,
```
Comment (alexsegura) [inline cube/Task.js:13]: Why did you remove the alias? Those columns should be renamed at some point (#535)

File: js/app/metrics/components/LogisticsDashboard.js [hunk 2]
```diff
@@ -22,6 +28,56 @@ const Dashboard = ({ cubejsApi, dateRange }) => {
         <ChartPanel title="Number of stores">
           <StoreCumulativeCount cubejsApi={ cubejsApi } dateRange={ dateRange } />
         </ChartPanel>
+        <div>
```
[the remark this answers is not in the document]
Comment (vladimir-8) [reply js/app/metrics/components/LogisticsDashboard.js]: it's a line break 🙈

File: cube/Task.js (hunk 1 above)
Comment (vladimir-8) [reply cube/Task.js:13]: I rename them when I create dimensions, so I thought it would be less confusion to have original names here

see https://github.com/coopcycle/coopcycle-web/pull/3107/files/abf8b28e867ea236875e15bc199d10f2486e07ec#diff-42bcc539c0025a538098703736d8be20df36864ca909db3ee39b3949c4fe4209R86

File: js/app/metrics/components/LogisticsDashboard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (vladimir-8) [reply js/app/metrics/components/LogisticsDashboard.js]: Screen Shot 2022-02-17 at 16 22 48

[the remark this answers is not in the document]
Comment (vladimir-8) [reply js/app/metrics/components/LogisticsDashboard.js]: my guess is that we probably don't need all these graphs and some of them could be modified or removed, but I don't know which ones are more useful, that why I'm asking for the feedback from cooperatives

File: cube/TaskDoneEvent.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+cube(`TaskDoneEvent`, {
+  sql: `SELECT * FROM public.task_event WHERE name='task:done'`,
```
Comment (alexsegura) [inline cube/TaskDoneEvent.js:2]: It's out of the scope of this pull request, but maybe having this info about when the task was completed in the `task_event` table makes things a little bit complicated. Maybe we could also have a `task.completed_at` column?

**Review by alexsegura — APPROVED** [review]
Thanks a lot @vladimir-8 for this huge pull request 😮 🤩
